Dockerfile

```diff
@@ -4,17 +4,13 @@ LABEL "maintainer"="Sviatoslav Sydorenko <wk+re-actors@sydorenko.org.ua>"
 LABEL "repository"="https://github.com/re-actors/gh-action-pypi-publish"
 LABEL "homepage"="https://github.com/re-actors/gh-action-pypi-publish"
 
-LABEL "com.github.actions.name"="pypi-publish"
-LABEL "com.github.actions.description"="Upload Python distribution packages to PyPI"
-LABEL "com.github.actions.icon"="upload-cloud"
-LABEL "com.github.actions.color"="yellow"
-
 ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
 
-ADD LICENSE.md /LICENSE.md
-
 RUN pip install --upgrade --no-cache-dir twine
 
-ENTRYPOINT ["twine"]
-CMD ["upload", "dist/*"]
+WORKDIR /app
+COPY ./twine-upload.sh /app
+
+RUN chmod +x /app/twine-upload.sh
+ENTRYPOINT ["/app/twine-upload.sh"]
```

Comment:
Did you do this to avoid env vars being logged?

Reply:
No the main reason was to do the mapping of the ENV vars from `INPUT_` to `TWINE_`